db/schema.rb

```diff
@@ -208,6 +208,7 @@
     t.datetime "locked_at"
     t.string   "locale",                 default: "es"
     t.boolean  "notifications",          default: true
+    t.boolean  "push_notifications",     default: true, null: false
```

Comment:
Do we need to add a data migration for the existing records?

Reply:
I believe the default makes it unnecessary.

Reply:
existing records will default to `true`